index.php

```diff
@@ -157,7 +154,7 @@ function init() {
 
         if (window.excimerData === undefined) {
             setLoading(true);
-            d3.json("json.php?<?php echo $_SERVER['QUERY_STRING'] ?>", function(error, data) {
+            d3.json("flamegraph.json.php?<?php echo $_SERVER['QUERY_STRING'] ?>", function(error, data) {
```

Comment:
using a raw QUERY_STRING feels like a security issue. We do want this to be generic, so maybe the better way is to use new moodle_url() and then use it to mutate the script and then ->out on it